frontend/src/download.js

```diff
@@ -107,19 +107,30 @@ $(document).ready(function() {
     fileReceiver.on('decrypting', isStillDecrypting => {
       // The file is being decrypted
       if (isStillDecrypting) {
-        console.log('Decrypting');
+        document.l10n
+          .formatValue('downloadDecryptingFile')
+          .then(downloadDecryptingFile => {
+            $('.progress-text').text(downloadDecryptingFile);
+          });
       } else {
-        console.log('Done decrypting');
         downloadEnd = Date.now();
       }
     });
 
     fileReceiver.on('hashing', isStillHashing => {
       // The file is being hashed to make sure a malicious user hasn't tampered with it
       if (isStillHashing) {
-        console.log('Checking file integrity');
+        document.l10n
+          .formatValue('downloadHashingFile')
+          .then(downloadHashingFile => {
+            $('.progress-text').text(downloadHashingFile);
+          });
       } else {
-        console.log('Integrity check done');
+        document.l10n
+          .formatValue('downloadNotification')
+          .then(downloadNotification => {
+            $('.progress-text').text(downloadNotification);
```

Comment:
This is just our existing "Your download has completed." messaging. Otherwise it seems to stick on the "Checking file integrity" message which sounds ominous.